accord-maelstrom/src/main/java/accord/maelstrom/Body.java

```diff
@@ -23,7 +24,7 @@
 
     public Body(Type type, long msg_id, long in_reply_to)
     {
-        this.type = type;
+        this.type = Objects.requireNonNull(type);
```

Comment:
no longer needed, was used to find where this was happening; can revert if desired

Reply:
It's not Guava so I don't mind, though I prefer annotations (and we can perhaps later use byte weaving or something to insert run-time asserts during tests)

Reply:
It _looks_ like Guava so I did originally mind a little :)